source/configure/smtp-email.rst

```diff
@@ -154,7 +154,7 @@ Search for ``554 5.7.1 error`` and
 Checking your SMTP server is reachable 
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
--  Attempt to telnet to the email service to make sure the server is reachable.
+-  Attempt to telnet to the email service to make sure the server is reachable.For reference checkout https://docs.microsoft.com/en-us/exchange/mail-flow/test-smtp-with-telnet?view=exchserver-2019 
```

Comment:
Thank you for adding a reference link to help readers understand how to complete this step. Given that this article is specific to Microsoft Exchange 2019 servers, it may be too specific a reference to provide that may not be applicable to a wide enough audience. @justinegeffen - What do you think?

Reply:
Agreed. This could cause the reader frustration by clicking through to an irrelevant article. However, all the links I've tested default to ending with "view=exchserver-2019" as that's the most recent version. 

I propose rewording this sentence to:

"For reference checkout https://docs.microsoft.com/en-us/exchange/mail-flow/test-smtp-with-telnet?view=exchserver-2019. If you're using an earlier version that 2019, select that version from the drop-down menu at the top of the left-hand navigation."

Reply:
updated .Thanks @justinegeffen